cmd/tdex/depositmarket.go

```diff
@@ -2,26 +2,55 @@ package main
 
 import (
 	"context"
+	"errors"
+	"sort"
+	"time"
 
+	"github.com/tdex-network/tdex-protobuf/generated/go/types"
+	"github.com/vulpemventures/go-elements/elementsutil"
+
+	"github.com/tdex-network/tdex-daemon/config"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-daemon/pkg/bufferutil"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer"
+	"github.com/tdex-network/tdex-daemon/pkg/trade"
+	"github.com/tdex-network/tdex-daemon/pkg/wallet"
 	pboperator "github.com/tdex-network/tdex-protobuf/generated/go/operator"
 	pbtypes "github.com/tdex-network/tdex-protobuf/generated/go/types"
+	"github.com/vulpemventures/go-elements/address"
+	"github.com/vulpemventures/go-elements/network"
+	"github.com/vulpemventures/go-elements/pset"
+	"github.com/vulpemventures/go-elements/transaction"
 
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	MinMilliSatPerByte = 150
+	CrawlInterval      = 3
+	MinBaseDeposit     = 50000
+	MinQuoteDeposit    = 50000
+)
+
 var depositmarket = cli.Command{
 	Name:  "depositmarket",
 	Usage: "get a deposit address for a given market or create a new one",
 	Flags: []cli.Flag{
 		&cli.StringFlag{
-			Name:  "base_asset",
-			Usage: "the base asset hash of an existent market",
+			Name:  "quote_asset",
+			Usage: "the quote asset hash of an existent market",
 			Value: "",
 		},
+		&cli.BoolFlag{
+			Name:  "no-fragment",
+			Usage: "disable utxo fragmentation",
+			Value: false,
+		},
 		&cli.StringFlag{
-			Name:  "quote_asset",
-			Usage: "the base asset hash of an existent market",
-			Value: "",
+			Name:  "explorer",
+			Usage: "explorer endpoint url",
+			Value: "http://127.0.0.1:3001",
 		},
```

Comment:
why is this needed? can't we get the explorer from config ad done in depositfee?